### ISSUE TITLE
New `ArgumentError at /node/version` issue

### ISSUE THREAD

**Issue by jameskirsop** [ytti/oxidized-web#173]
I'm having an issue similar to #38 but it seems to be manifiesting its self slightly differently.

For nodes within a new group I've just created, download and version history doesn't work:

```
ArgumentError at /node/version
wrong number of arguments (0 for 1+)
```

My GET args are ```CAS/Netscaler```. The nodes causing this issue indicate an OK status in the `/nodes` list view.

All other groups seem to work OK, it's just this one named 'CAS'. The non-functioning nodes within this group are either (currently) Netscaler or ProCurve models and are stored in Oxidized using git.

**Comment by jameskirsop** [comment]
This can probably be closed. Turns out there was a problem with the model fetching config for the device in this group (indicating it had success when in fact it hadn't) and as a result, no git file existed for the group.

It would probably be nice to have the web UI handle this a bit nicer with some pointers to what to check, but not important given it's likely a rare circumstance.

**Comment by didawubo1** [comment]
I also encountered the same problem, did you solve it

**Comment by wk** [comment]
This should be resolved in version 0.28.0.
